qaframework-bdd-tests/cypress.json

```diff
@@ -6,14 +6,14 @@
   ],
   "video": true,
   "defaultCommandTimeout": 20000,
-  "baseUrl": "https://o3.openmrs.org/openmrs/spa",
+  "baseUrl": "https://openmrs-spa.org/openmrs/spa",
   "env": {
-    "API_BASE_URL": "https://o3.openmrs.org/openmrs/ws/rest/v1",
-    "ADMIN_USERNAME": "admin",
+    "API_BASE_URL": "https://openmrs-spa.org/openmrs/ws/rest/v1",
+    "ADMIN_USERNAME": "spa_user",
     "ADMIN_PASSWORD": "Admin123",
-    "ADMIN_UUID": "82f18b44-6814-11e8-923f-e9a88dcb533f",
-    "DEFAULT_LOCATION_UUID": "ba685651-ed3b-4e63-9b35-78893060758a",
-    "DEFAULT_LOCATION_NAME": "Inpatient Ward",
+    "ADMIN_UUID": "8f24ac87-2d71-4574-991e-97803e6f8095",
+    "DEFAULT_LOCATION_UUID": "6351fcf4-e311-4a19-90f9-35667d99a8af",
+    "DEFAULT_LOCATION_NAME": "Registration Desk",
```

Comment:
are these changes part of the ticket requirement?

Reply:
How did these get here?...lemi find out why

Reply:
@mherman22  what is your finding on the guys [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/313#discussion_r781457408)?

Reply:
> 

I did a squash that I should not have done in the first place. So I guess having those guys up there is part of the complications. I'll look into it

Reply:
@mherman22  did you finally looked into it.